.dockerignore

```diff
@@ -0,0 +1,6 @@
+.dockerignore
+Dockerfile
+.gitignore
+LICENSE
+README.md
+suisa_sendemeldung.conf
```

Comment:
This shouldn't be in the ignore as it wont get added to the build context and we absolutely want it to be in the image.

Otherwise I think the ignores are fine.